src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelector.js

```diff
@@ -163,27 +217,14 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
         const { onChange } = this.props;
 
         if (internalEditorState) {
-            this.setState(
-                {
-                    internalEditorState: nextEditorState,
-                },
-                () => {
-                    if (onChange) {
-                        onChange(nextEditorState);
-                    }
-                    this.checkValidity();
-                },
-            );
-        } else {
-            if (onChange) {
-                onChange(nextEditorState);
-            }
-            this.checkValidity();
+            this.setState({ internalEditorState: nextEditorState }, () => onChange && onChange(nextEditorState));
```

Comment:
Can we default `onChange` to `noop` while we're at it? Then it can just be called at the very bottom of the method outside of the conditions.